index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Mobile-First Portfolio | Home</title>
+    <link rel="stylesheet" href="styles.css">
+    <!-- Boxicon -->
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <!-- Google Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <!-- Start of Header Container -->
+    <header>
+        <nav class="navbar">
+            <div class="container navbar-container">
+                <a href="index.html" class="logo">Sri Sudha Suryadevara</a>
+                <ul class="nav-items">
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+                <i class='bx bx-menu'></i>
+            </div>
+        </nav>
+    </header>
+
+    <!-- start of Main Container-->
+    <main>
+        <section class="hero">
+            <div class="container">
+                <div class="hero-text">
+                    <h1>I’m Sri Sudha Suryadevara.
+                        Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some
+                        of my
+                        work and experience! If you like what you see and have a project you need coded, don’t hesitate
+                        to
+                        contact me.</p>
+                    <a href="#" class="link-text">LET’S CONNECT</a>
+                    <ul class="social">
+                        <li><a href="twitter.com"><img src="Images/icons/Twitter.png" alt="twitter-img"></a></li>
+                        <li><a href="linkedin.com"><img src="Images/icons/Linkedin.png" alt="linkedin-img"></a></li>
+                        <li><a href="medium.com"><img src="Images/icons/Medium.png" alt="Medium-img"></a></li>
+                        <li><a href="github.com"><img src="Images/icons/Git.png" alt="Github-img"></a></li>
+                        <li><a href="AngelList.com"><img src="Images/icons/AngelList.png" alt="AngelList-img"></a></li>
+                    </ul>
+                </div>
```

Comment:
- [ Required ] I think it would be better if you align the `.hero-text` div using `flexbox` as suggested by the previous reviewer, this will make it easy for you to adapt your design to different screen sizes. I would like to suggest that you refactor this piece of CSS in your `styles.css` as follows. This will make your design to match the `margins` and `backgrounds` in the Figma design.

```CSS
.hero-text {
    display: flex;
    flex-direction: column;
    margin-top: 120px;
    justify-content: flex-start;
}
```

## Screenshot
| Your design | Figma design |
|-----------------|--------------------|
|![Screenshot from 2023-07-08 02-27-35](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/16392046/f6a6efa0-a941-475d-b9bb-ee3db5d14c1f) | ![](https://github.com/microverseinc/curriculum-html-css/raw/main/portfolio/images/m1_setup/headline-mobile.png) |

| After applying changes |
|--------------------------------|
|![Screenshot from 2023-07-08 02-32-23](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/16392046/d4f90fcc-f38d-4766-88d5-e99b35cb7da4)|